src/core/aws-messages.ads

```diff
@@ -204,6 +204,10 @@ package AWS.Messages is
    function Reason_Phrase (S : Status_Code) return String;
    --  Returns the reason phrase for the status code S, see [RFC 2616 - 6.1.1]
 
+   function With_Body (S : Status_Code) return Boolean is
```

Comment:
expression function definition should be in the private part. only the spec visible.

Reply:
Done